app/views/layouts/application.html.erb

```diff
@@ -16,7 +16,7 @@
 
 <%= render 'govuk_publishing_components/components/layout_for_admin',
   environment: GovukPublishingComponents::AppHelpers::Environment.current_acceptance_environment,
-  browser_title: yield(:title) do %>
+  browser_title: (yield(:browser_title).presence || yield(:title)) + " - GOV.UK" do %>
```

Comment:
`" - GOV.UK" ` was removed in a previous commit, as it's now added by the component.

Reply:
Ah, sorry - didn't reload my rails. That's better!